ramalama/model.py

```diff
@@ -171,10 +174,14 @@ def setup_container(self, args):
             conman_args += ["--device", "/dev/kfd"]
 
         for k, v in get_env_vars().items():
-            conman_args += ["-e", f"{k}={v}"]
-
+            # Special case for Cuda
```

Comment:
I think we should remove the else here and the env var in all cases, a use might set CUDA_VISIBLE_DEVICES to run a specific GPU and llama.cpp should be able to process that env var.

Reply:
The else should be gone here right?